### ISSUE TITLE
[cmake] make tarball gives directories different names (Trac #2373)

### ISSUE THREAD

**Issue by icecube-trac** [icecube-trac/tix2#2365]
For some reason the normal build directories put simprod-scripts in a directory called `simprod-scripts` with a dash, but make tarball creates a directory called `simprod_scripts` with an underscore. This messes up a lot of my scripts. All other directory names remain unaffected.

Migrated from https://code.icecube.wisc.edu/ticket/2373
```json
{
    "status": "closed", 
    "changetime": "2020-06-24T12:31:42", 
    "description": "For some reason the normal build directories put simprod-scripts in a directory called `simprod-scripts` with a dash, but make tarball creates a directory called `simprod_scripts` with an underscore. This messes up a lot of my scripts. All other directory names remain unaffected.", 
    "reporter": "kjmeagher", 
    "cc": "", 
    "resolution": "fixed", 
    "_ts": "1593001902142004", 
    "component": "cmake", 
    "summary": "[cmake] make tarball gives directories different names", 
    "priority": "normal", 
    "keywords": "", 
    "time": "2019-11-08T16:44:28", 
    "milestone": "Autumnal Equinox 2020", 
    "owner": "nega", 
    "type": "defect"
}
```

**Comment by icecube-trac** [comment]
Trac update at 20191126T17:27:54: 
* olivas changed owner from "" to "nega"
* olivas changed status from "new" to "assigned"

**Comment by icecube-trac** [comment]
Trac update at 20191220T17:57:55: 
* olivas changed status from "assigned" to "accepted"
* olivas changed owner from "nega" to "olivas"

**Comment by icecube-trac** [comment]
Trac update at 20191220T18:43:06: 
* olivas commented:

> Milestone Winter Solstice 2019 deleted


* olivas changed milestone from "Winter Solstice 2019" to "Vernal Equinox 2020"

**Comment by icecube-trac** [comment]
Trac update at 20200224T17:53:32: nega changed owner from "olivas" to "nega"

**Comment by icecube-trac** [comment]
Trac update at 20200304T17:56:46: nega commented:

> can't build tarball because `install_python.py` was removed from `simprod-scripts` in Nov 2019, and now `simprod-scripts` internal targets fail.

**Comment by icecube-trac** [comment]
Trac update at 20200305T20:16:00: 
* nega changed status from "accepted" to "closed"
* nega changed resolution from "" to "fixed"
* nega commented:

> In changeset 179255:
> 
> ```text
> #!CommitTicketReference repository="IceCube" revision="179255"
> fix project name in i3_project()
> 
> set the proper project name. this was causing internal conflicts because the
> directory name in the tarball was different from the build and src
> directories. 
> 
> closes: #2365
> ```

**Comment by icecube-trac** [comment]
Trac update at 20200624T12:31:42: 
* olivas commented:

> Milestone Vernal Equinox 2020 deleted


* olivas changed milestone from "Vernal Equinox 2020" to "Autumnal Equinox 2020"
